app/temporal/utils.ts

```diff
@@ -37,7 +40,10 @@ export const retryWorkflow = async <T>(
       }
       lastError = err;
       if (i < options.maxRetries - 1) {
-        await sleep(options.retryIntervalMs);
+        await sleep(retryIntervalMs);
+        if (options.isExponential ?? false) {
+          retryIntervalMs *= 2;
```

Comment:
Please cap the maximum delay, otherwise a temporary failure in an agent might cause an extreme delay

Reply:
The delay is already capped because the function requires the `maxRetries` argument.